server/src/http_server/pages/blog/mod.rs

```diff
@@ -171,10 +171,11 @@ pub(crate) async fn post_get(
 
     let cover_photo = markdown.ast.0.cover_photo();
 
-    let html = match markdown
-        .ast
-        .into_html(&state.app, &state.markdown_to_html_context)
-    {
+    // creating some context for the article_path
```

Comment:
`context` vat attempts to clone the existing `markdown-to-html` context and `updates` it with the current `article's path`